packages/feedback/.eslintrc.js

```diff
@@ -0,0 +1,42 @@
+// Note: All paths are relative to the directory in which eslint is being run, rather than the directory where this file
+// lives
+
+// ESLint config docs: https://eslint.org/docs/user-guide/configuring/
+
+module.exports = {
+  extends: ['../../.eslintrc.js'],
+  overrides: [
+    {
+      files: ['src/**/*.ts'],
+      rules: {
+        '@sentry-internal/sdk/no-unsupported-es6-methods': 'off',
```

Comment:
probably want to turn this back on, if this should not depend on replay only. Replay is only built for ES6, but the rest of the SDK is also built for ES5. If this should not depend on replay, let's remove this line (and avoid using these functions)